examples/events.py

```diff
@@ -32,6 +32,9 @@
     await container.start(config)
     print(f"=> created and started container {container._id[:12]}")
 
+    container = yield from docker.containers.create_or_replace(config=config, name='testing')
+    yield from container.start()
+
```

Comment:
`create_or_replace` is already there a few lines above.
`yield from` is not allowed here as we use Python 3.6 async/await syntax.
Looks like a mistaken merge.